docs/user/OIDC/OAuth2OIDC-keycloak.md

```diff
@@ -1,9 +1,310 @@
 # Configuring Keycloak as an OIDC provider
+(created May 17th 2021)
```

Comment:
Not sure if we should add the date here, as it is kind of redundant (this info is already in git) and we don't have it in the rest of the docs.

Reply:
i will remove the date, and instead include the versions of the charts i used (easier for users)